localgov_publications.install

```diff
@@ -5,17 +5,37 @@
  * Install, update and uninstall functions for the LocalGov Publications module.
  */
 
+use Drupal\filter\Entity\FilterFormat;
+
 /**
  * Implements hook_install().
  */
 function localgov_publications_install() {
+
+  // Add our localgov_publication_page content type to
+  // book.settings.allowed_types. This lets editors create publication pages.
   $config = \Drupal::configFactory()->getEditable('book.settings');
   $allowed_types = $config->get('allowed_types');
   $allowed_types[] = 'localgov_publication_page';
   $config->set('allowed_types', $allowed_types);
   $config->save();
+
+  // Add our localgov-publication-cover-page-alias to
+  // pathauto.settings.safe_tokens. This prevents double escaping in the
+  // resulting URL.
   $path_auto_config = \Drupal::configFactory()->getEditable('pathauto.settings');
   $safe_tokens = $path_auto_config->get('safe_tokens');
   $safe_tokens[] = 'localgov-publication-cover-page-alias';
   $path_auto_config->set('safe_tokens', $safe_tokens)->save();
+
+  // Add heading_id_filter from the auto_heading_ids module to the wysiwyg
+  // filter format. This is required for the ToC block to work consistently.
+  /** @var \Drupal\filter\Entity\FilterFormat $wysiwygFormat */
+  $wysiwygFormat = FilterFormat::load('wysiwyg');
+  if ($wysiwygFormat) {
```

Comment:
`if ($wysiwygFormat instanceof FilterFormatInterface)` and add at the top `use Drupal\filter\FilterFormatInterface`

Reply:
Good plan. Done.